index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0039)https://skycocker.github.io/chromebrew/ -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link rel="shortcut icon" href="brew.png"" type="image/png">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="The missing package manager for Chrome OS">
     <meta name="author" content="Michal Siwek">
-    <link rel="shortcut icon" href="favicon.png">
+    <link rel="shortcut icon" href="https://skycocker.github.io/chromebrew/favicon.png">
```

Comment:
The fully qualified domain is not needed.

Reply:
WHoops

Reply:
but that is the icon...

Reply:
You can still remove the fully qualified domain here.

Reply:
Just restore to the way it was before you changed it.

Reply:
Can you fix this please?